src/tool/cli.rs

```diff
@@ -43,9 +43,9 @@ impl Cli {
             env::current_dir()?.display()
         );
         if let Some(path) = &self.path {
-            info!("Going to update working directory to to '{path}'");
+            info!("Going to update working directory to to '{path:?}'");
             env::set_current_dir(path)
-                .with_context(|| format!("Failed to set current dir to: '{path}'"))?;
+                .with_context(|| format!("Failed to set current dir to: '{path:?}'"))?;
```

Comment:
Do you think it is worth the extra verbosity to use the display version of the path instead of the debug one? To be clear Display is not implemented for path but it does have a display function, that's the one I'm talking about. Not saying we should I'm just asking what you think now that we've decided to keep it as a `PathBuf`

Reply:
Does the `:?` make any difference?

Reply:
When you asked I realized I don't actually know. They are not the same. Display omits the non-unicode characters while debug escapes them. 

https://doc.rust-lang.org/std/path/struct.Path.html#method.display

For our use case here I guess we can go with debug as you have it already.